.dockerignore

```diff
@@ -1,2 +1,6 @@
 .cosa
+_kola_temp
+docs
+ci
```

Comment:
I think this is breaking:
https://github.com/containers/bootc/blob/fad73f26a49eec58b3a150dd3579e9ed7d3a92d6/ci/Dockerfile.ci#L7

Reply:
Ah right, thanks for debugging this!  It's not actually important so dropped